docs/examples/commercial_aircraft/commercial_aircraft.ipynb

```diff
@@ -0,0 +1,674 @@
+    "|mass\\_empty    |aircraft empty mass (kg)   |input          |\n",
+    "|mass\\_payload  |payload mass (kg)          |input          |\n",
+    "|mass\\_fuel     |fuel mass (kg)             |input          |\n",
+    "|mass\\_total    |total aircraft mass (kg)   |output         |\n",
+    "|W\\_total       |total aircraft weight (N)  |output         |\n",
+    "\n",
+    "### Dynamic Pressure\n",
+    "\n",
+    "The _DynamicPressureComp_ computes the dynamic pressure from\n",
+    "true airspeed and atmospheric density.\n",
+    "\n",
+    "\\begin{align}\n",
+    "  q &= \\frac{1}{2} \\rho TAS^2\n",
+    "\\end{align}\n",
+    "\n",
+    "|Name  |Description                      |Input or Output|\n",
+    "|------|---------------------------------|---------------|\n",
+    "|TAS   |true airspeed (m/s)              |input          |\n",
+    "|rho   |atmospheric density (kg/m\\*\\*3)  |input          |\n",
+    "|q     |dynamic pressure (Pa)            |output         |\n",
+    "\n",
+    "### Aerodynamics\n",
+    "\n",
+    "The aerodynamics group computes the aerodynamic coefficients and forces\n",
+    "on the vehicle. It consists of an interpolation component which outputs\n",
+    "lift, drag, and moment coefficients as a function of Mach number, angle\n",
+    "of attack, altitude, and tail rotation angle. A second component then\n",
+    "uses these coefficients, along with dynamic pressure and aerodynamic\n",
+    "reference area, to compute the lift and drag forces on the vehicle.\n",
+    "\n",
+    "The aerodynamics group resides within the flight equilibrium group. As\n",
+    "that group iterates to find the combination of thrust coefficient, angle\n",
+    "of attack, and tail rotation angle, aerodynamics needs to update the\n",
+    "values of the interpolated coefficients and resulting forces.\n",
+    "\n",
+    "Organizationally speaking, we logically could have put the dynamic\n",
+    "pressure component within the aerodynamics group. However, since that\n",
+    "group doesn't need to be updated with changes in alpha and tail angle,\n",
+    "it's more efficient to leave it outside of flight equilibrium group.\n",
+    "\n",
+    "  |Name   |Description                |Input or Output|\n",
+    "  |-------|---------------------------|---------------|\n",
+    "  |mach   |mach number                |input          |\n",
+    "  |alt    |altitude (m)               |input          |\n",
+    "  |alpha  |angle of attack (deg)      |input          |\n",
+    "  |eta    |tail rotation angle (deg)  |input          |\n",
+    "  |CL     |lift coefficient           |output         |\n",
+    "  |CD     |drag coefficient           |output         |\n",
+    "  |CM     |moment coefficient         |output         |\n",
+    "  |L      |lift force (N)             |output         |\n",
+    "  |D      |drag force (N)             |output         |\n",
+    "\n",
+    "!!! note \"Note\"\n",
```

Comment:
This note should use the note directive.

Reply:
Fixed